packages/ansifilter.rb

```diff
@@ -3,28 +3,25 @@
 class Ansifilter < Package
   description 'Ansifilter parses common ANSI codes to remove them or to convert them to another colored text file format (HTML, TeX, LaTeX, RTF, Pango or BBCode).'
   homepage 'http://andre-simon.de/doku/ansifilter/en/ansifilter.php'
-  version '2.10'
+  version '2.20'
   license 'GPL-3+'
   compatibility 'all'
-  source_url 'http://www.andre-simon.de/zip/ansifilter-2.10.tar.bz2'
-  source_sha256 '23d2cf439d4ed4fbec8050b2826d61c244694ce06aaf8ca7d0ec1016afebee3f'
-  binary_compression 'tar.xz'
+  source_url 'https://gitlab.com/saalen/ansifilter.git'
+  git_hashtag version
+  binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: 'd5a8d33520c76aeb3f13067f169300d4b959a78ef352331b14a1b70db9f44f5f',
-     armv7l: 'd5a8d33520c76aeb3f13067f169300d4b959a78ef352331b14a1b70db9f44f5f',
-       i686: 'b312b46ae42dfb60620e6f62caa5aae83bfe259e01094d84f2dd949a137eedf2',
-     x86_64: 'af4c1a4f1b9fddd1efa5dabded1664880bd25fc17b52290af956d4d5d7dedf5c'
+    aarch64: 'da375a75732ae5d4534a7278c4d40dd26d89bdf9aa3a8f7eed1f9ce38ee65905',
+     armv7l: 'da375a75732ae5d4534a7278c4d40dd26d89bdf9aa3a8f7eed1f9ce38ee65905',
+       i686: 'b42d604aa940d8ebf298958fb1a4b93b3d2dcf29b7900783c914947878e9665f',
+     x86_64: '12f6c518b5c1cf12192cde003ca0df4ce16a5bbf72dd23bddd3c34cb743cbbd4'
```

Comment:
Were these binaries generated in the lower glibc containers?

Reply:
yeah